src/display/canvas.js

```diff
@@ -2480,6 +2550,9 @@ const CanvasGraphics = (function CanvasGraphicsClosure() {
     }
 
     paintInlineImageXObject(imgData) {
+      if (imgData) {
+        return;
+      }
```

Comment:
This definitely looks weird (and likely wrong); won't this cause a whole bunch of errors during tests?
Please note https://github.com/mozilla/pdf.js/wiki/Contributing#4-run-lint-and-testing

Reply:
Ah damn, that was some debugging code left-overs.